pkg/cli/cmds/etcd_snapshot.go

```diff
@@ -0,0 +1,38 @@
+package cmds
+
+import (
+	"github.com/rancher/k3s/pkg/version"
+	"github.com/urfave/cli"
+)
+
+const EtcdSnapshotCommand = "etcd-snapshot"
```

Comment:
We're building this to be used as `k3s etcdsnapshot` right? The binaries are called `k3s-etcdsnapshot` so shouldn't this be:
```suggestion
const EtcdSnapshotCommand = "etcdsnapshot"
```